cmd/private-org-sync/main.go

```diff
@@ -461,34 +461,70 @@ func main() {
 		failOnNonexistentDst: o.failOnNonexistentDst,
 	}
 
-	var syncErrors []error
+	var errs []error
+	locations := make(map[location]struct{})
+
+	if whitelistErrors := getWhitelistedLocations(o.WhitelistOptions.WhitelistConfig.Whitelist, syncer.git, locations); len(whitelistErrors) > 0 {
+		errs = append(errs, whitelistErrors...)
```

Comment:
This doesn't need to be in an if; both an empty slice and a nil slice can be appended like this.